ana-agudelo2/tarea-2/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <title>Ana Agudelo - Tarea 2</title>
+        <meta charset="UTF-8" />
+        <meta name="viewport" content="width=device-width" />
+        <link rel="stylesheet" href="index.css"/>
+    </head> 
+    
+    <body>
+        <main>
+            <section class="grid-container">
+                <li class="item1">August 2022</li>
```

Comment:
@ana19051 , li tags should be under ul tags https://www.w3schools.com/html/html_lists.asp

Reply:
@emolinaab Ok, I solved the problem and I learned to use the ul and li. Thanks a lot.